pkg/local_object_storage/metabase/cleanup.go

```diff
@@ -1,108 +0,0 @@
-package meta
-
-import (
-	"bytes"
-	"strings"
-
-	"go.etcd.io/bbolt"
-)
-
-// CleanUpPrm groups the parameters of CleanUp operation.
-type CleanUpPrm struct{}
-
-// CleanUpRes groups resulting values of CleanUp operation.
-type CleanUpRes struct{}
-
-// CleanUp removes empty buckets from metabase.
-func (db *DB) CleanUp(prm *CleanUpPrm) (res *CleanUpRes, err error) {
```

Comment:
why deleted? i think it completes the metabase's API (there are some things that are not used for now but not deleted yet in `pkg`, never thought that it must be deleted)

Reply:
Unused, can be returned with single revert.